HandyImage.user.js

```diff
@@ -796,17 +795,19 @@ function makeworld()
 	{
 	case "gist.github.com":
 		if(document.body){i=1;cfg();}break;
-	case "vvcap.net":
 	case "simplest-image-hosting.net":
 	case "hostimage.ru":
 	case "imagebin.ca":
 	case "imgchili.net":
-	case "pixhst.com":
 	case "adultimages.xyz":
-	case "depic.me":
 		i = q('img');
+		if(i)
```

Comment:
you should check 'if(i & i.src)' else there might be errors, + there is a link to full image on the page, no need for replacing...